include/villas/nodes/websocket.hpp

```diff
@@ -25,7 +26,7 @@ namespace villas {
 namespace node {
 
 /* Forward declarations */
-class NodeCompat;
+// class NodeCompat;
```

Comment:
Please try to keep the forward declaration here as it reduces the amount of code which needs to get recompiled if `node_compat.hpp` changes.

The proper way of fixing this, to remove the code in the header which relies on the full declaration of NodeCompat to the .cpp file.

Reply:
fixed